platformio.ini

```diff
@@ -85,7 +85,7 @@ extra_scripts = ${env.extra_scripts}
 ;~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~;
 [esp32git]
 extends = esp32
-build_flags = ${esp32.build_flags}
+build_flags = ${esp32.build_flags} -mtext-section-literals
```

Comment:
For this commit, you wrote:
> Added compiler option that does not include unused function / modules in the output image

However, the [documentation page](https://gcc.gnu.org/onlinedocs/gcc/Xtensa-Options.html#index-mtext-section-literals) doesn't indicate this flag affects whether an unused function or module is in the final output image.

Can you help me understand the source that suggested this happens via the use of this parameter?